src/FocusTrap.ts

```diff
@@ -72,7 +72,7 @@ const FocusTrap: FocusTrapComponent = {
             this.$el,
             {
               escapeDeactivates: this.escapeDeactivates,
-              allowOutsideClick: () => this.allowOutsideClick,
+              allowOutsideClick: this.allowOutsideClick,
```

Comment:
We still need a function to handle reactive values

```suggestion
              allowOutsideClick: (e) => typeof this.allowOutsideClick === 'function' ? this.allowOutsideClick(e) : this.allowOutsideClick,
```

Reply:
Right you are - I was forgetting about the fact that the prop can obviously dynamically change..